### PR TITLE
Update svg.js to use npm auto-update

### PR DESCRIPTION
**Pull request by Fuzzyma**
Pull request for issue: #13112 

**This libraray is already available on cdnjs. So all checks are fullfilled.**

As requested, I opened a PR to change the svg.js endpoint to npm.

### CODE REVIEW

File: ajax/libs/svg.js/package.json [hunk 1]
```diff
@@ -10,22 +10,19 @@
...
   "repository": {
     "type": "git",
     "url": "https://github.com/svgdotjs/svg.js/"
   },
+  "npmName": "@svgdotjs/svg.js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "*"
```
Comment (extend1994) [inline ajax/libs/svg.js/package.json]: Hi, @Fuzzyma I'm wondering `svg.node.js` is for node environment, is that so?

Comment (Fuzzyma) [reply ajax/libs/svg.js/package.json]: Yes thats true. Shall I exclude it? If so, is `!(.*)node(.*)` sufficiant?

Comment (extend1994) [reply ajax/libs/svg.js/package.json]: Maybe `**/!(*.node.*)`? It grabs all files except for the ones with the filename`.node.` recursively.

Comment (Fuzzyma) [reply ajax/libs/svg.js/package.json]: Alright, will use that. Thank you!

**Comment by Fuzzyma** [comment]
To rebase I would need to pull at least the spare repo. However pulling 2Gigs of files is not possible for me.
I did all changes directly inline on github so I  dont need to pull the repo. Can you rebase for me? (since I only changed one folder alone, there should be no rebase required anyway)

**Pull request merged**